openshift/templates/dbbackup.dc.json

```diff
@@ -58,7 +76,23 @@
       },
       "spec": {
         "strategy": {
-          "type": "Recreate"
+          "type": "Recreate",
+          "recreateParams": {
+            "pre": {
+              "failurePolicy": "Retry",
+              "execNewPod": {
+                "containerName": "flyway-migration",
+                "command": [
+                  "bash",
+                  "-c",
+                  "$FLYWAY_HOME/scripts/run_migrations.sh"
+                ]
+              }
+            },
+            "maxSurge": "50%",
+            "maxUnavailable": 0,
+            "timeoutSeconds": 1200
+          }
```

Comment:
the dbbackup pod is long-living, meaning it has no need to redeploy on sql changes, not sure this will keep the datastore db up to date without intervention.

Reply:
(discussed offline) :)